css/main.css

```diff
@@ -46,6 +46,14 @@ pre.highlight code,
   padding: 0 5px;
 }
 
+.post h2 {
+  font-size: 150%;
+}
+
+.post h3 {
+  font-size: 125%;
+}
+
```

Comment:
The headings were too big to my liking.

Have not checked how things look if one starts using `h4` or `####` for headings